### ISSUE TITLE
Sonoff-Tasmota KNX Features

### ISSUE THREAD

**Issue by thomas-varelas** [TasmoAdmin/TasmoAdmin#252]
I am currently working with Tasmota KNX and I would like to ask how can I get the KNX parameters from the device?   Are there any functions similar to those on Sonoff.php which I must use to get the current KNX configuration of the device?

**Comment by reloxx13** [comment]
its more in js.

are those knx infos in the status 0?
could you post it?

**Comment by ascillato** [comment]
The KNX config is not in status 0. It can be access from the webmenu in Tasmota or by commands

**Comment by reloxx13** [comment]
so @thomas-varelas want those params on the device config page? i can do that

**Comment by ascillato** [comment]
For each parameter is a different command. May be will be easiest for TasmoAdmin if I add a new status in status 0 with all those parameters

**Comment by thomas-varelas** [comment]
Yes, it would be perfect! My primary goal is to create an easy interface where the user will be able to configure knx and add a 2 way or a 3 way switch. So if those knx related commands were added to status 0 that would be easy.

**Comment by reloxx13** [comment]
@ascillato thats not neccessery, i do this already with mqtt.

would be enough to have a 1/0 in status 0 for knx is enabled or not, if its enabled i have to call all commands to get the infos.

**Comment by ascillato** [comment]
Ok. Great

**Comment by ascillato** [comment]
It is already in status 0, in features.

**Comment by thomas-varelas** [comment]
So are we ok with this? Should I close the issue?

**Comment by reloxx13** [comment]
noo let it open, ill close it if im done.

**Comment by reloxx13** [comment]
Triage note: this is still only partially supported. The current code can detect KNX-related feature flags in device options, but there is still no dedicated KNX configuration/editor experience in the device UI for the broader workflow requested in this thread. Keeping this open with low priority.
